packages/nimble-components/src/table/specs/table-columns-hld.md

```diff
@@ -250,38 +313,45 @@ public class TableColumnPositiveNegativeNumber extends TableColumn<TableColumnPo
         return [valueKey];
     }
 
-    public readonly cellStyles: ElementStyles =
-        css`
-            .good {
-                color: green;
-            }
-
-            .bad {
-                color: red;
-            }
-        `;
-
-    public readonly cellTemplate: ViewTemplate<TableCellState<TableColumnPositiveNegativeNumberCellData>> =
-        html<TableCellState<TableColumnPositiveNegativeNumberCellData>`
-            <nimble-text-field
-                class="${x => isPositive(x.data.value) ? "good" : "bad"}"
-                readonly="true"
-                value="${x => x.data.value}"
-            >
-            </nimble-text-field>
-        `;
-
     public validateCellData(cellData: TCellRecord): void {
         if (typeof(cellData['value']) !== 'number') {
             throw new Error('Type for cellData is incorrect!');
         }
     }
 }
+
+class PositiveNegativeNumberCellElement extends BaseCellElement<TableColumnPositiveNegativeNumberCellData> {
+    @volatile
+    public get textFieldCssClass(): string {
+        return this.cellRecord.value > 0 ? 'good' : 'bad';
+    }
+}
+const positiveNegativeNumberCellElement = PositiveNegativeNumberCellElement.compose({
+    baseName: 'table-cell-element-positive-negative-number',
+    template: html<PositiveNegativeNumberCellElement>`
+        <nimble-text-field
+            class="${x => x.textFieldCssClass}"
+            readonly="true"
+            value="${x => x.cellRecord.value}"
+        >
+        </nimble-text-field>`,
+    styles: css`
+        .good {
+            color: green;
+        }
+
+        .bad {
+            color: red;
+        }
+    `
+});
 ```
 
-Finally, here is a column element that allows a user to register a callback for a click event on a button inside the cell template:
+Finally, here is a column element that allows a user to register a callback for a click event on a button inside the cell template:  
+**TODO: Update the following example based on custom-element-in-cell changes**
```

Comment:
Here's a comment to use to track this TODO :).

Reply:
Updated the last example